### PR TITLE
Improved cursors on game over text and buttons

### PR DESCRIPTION
**Pull request by foolmoron**

### CODE REVIEW

**Comment by niccokunzmann** [comment]
With #277, you added css to the svg file. If you would like to avoid conflict, it seems you can add the same css to the index.css file. It seems to be included in the first lines. If not, including a CSS file surely would be of help. 

https://raw.githubusercontent.com/fossasia/flappy-svg/f8c473dc6212839991308c10a51d99ed98a95073/flappy.svg

    <?xml-stylesheet type="text/css" href="index.css" ?>

**Comment by foolmoron** [comment]
Okay I'll update my PRs. I was trying to link an external CSS file but it doesn't work with embedded SVG like what you're doing. You have to do inline CSS. But I think if you embed with an object tag instead, it could work. I'll try it out!

**Comment by foolmoron** [comment]
Actually the only issue was the `index.css` link in the svg had to be `style/index.css`. I updated my PRs so they should be okay to merge now.

File: style/index.css [hunk 1]
```diff
@@ -6,4 +6,25 @@
...
+	opacity: 0.7;
+}
+g#layer4 image:hover {
+	opacity: 1.0;
+}
+
+tspan {
+   cursor: default;
+}
+#g17141, #g17141 tspan {
```
Comment (niccokunzmann) [inline style/index.css]: One more question:
These ids are ids which are auto-generated. You can give these objects Ids yourself.
In this case, they will probably stay in further edits.

I am merging this as it works for now.

**Pull request merged**